.github/workflows/release-cnab.yaml

```diff
@@ -22,11 +22,12 @@ jobs:
           helm repo add incubator https://charts.helm.sh/incubator
           helm repo add elasticsearch https://helm.elastic.co  
           helm repo add grafana https://grafana.github.io/helm-charts
+          helm repo add ollama https://otwld.github.io/ollama-helm
 
       - name: Install Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v4.2.0
```

Comment:
Is this change required for ollama?

Reply:
yep, otherwise parser fails. The version of helm is too old, so I just bumped it everywhere to have it consistent